packages/web-components/src/progress-bar/progress-bar.options.ts

```diff
@@ -1,5 +1,4 @@
-import type { StaticallyComposableHTML, ValuesOf } from '../utils/index.js';
-import { ProgressBar } from './progress-bar.js';
```

Comment:
nope, but I agree that it can be a distortion for a reviewer, I tend to remove warnings and dead code where it's possible while working on the code, so at least a comment in PR in this case could make it better  🙂